source/Mlos.Core/MlosContext.h

```diff
@@ -126,7 +133,7 @@ class InternalMlosContextInitializer
 
     InternalMlosContextInitializer& operator=(const InternalMlosContextInitializer&) = delete;
 
-public:
+private:
```

Comment:
@grlap for visibility

Something I noticed while reading through.  Seemed inappropriate for these fields to be public, but it's probably also inappropriate to include this change in this PR.

Reply:
Backed this out - it doesn't belong here.  Can take it up separately later on.